### PR TITLE
Sub users on card

### LINKED ISSUE #64: **subscribed users on cards**
- display subscribed users on cards in notion last seen format

### PR DESCRIPTION
**Pull request by andrewvenson**
- Created view for changing profile image
- view changes profile image in state and firebase on changing avatar press
- display current users avatar on crawl cards upon subscription, limit is 5 and then will display the rest of the users in num format
- unsubscribe takes off avatar from card in question

closes #64

### CODE REVIEW

**Comment by andrewvenson** [comment]
If you change your avatar image, you'll have to unsub and then sub from a card to see your new profile changes on crawl card. I'll be adding like 30 users today for testing

File: pubio/components/CarouselCards.js [hunk 1]
```diff
@@ -1,123 +1,45 @@
...
+      date={item.date}
+      info={item.info}
+      imageURL={item.imageURL}
+      crawlIndex={index}
+    />
+  );
 }
 
-const styles = StyleSheet.create({});
+export default function CarouselCards() {
```
Comment (tan-x) [inline pubio/components/CarouselCards.js:21]: Why you hating on classes??

File: pubio/components/Cards.js [hunk 2]
```diff
@@ -12,12 +12,77 @@ import {
...
+          return (
+            <Avatar.Image
+              key={index}
+              style={{
+                borderWidth: 0.7,
+                borderRadius: 50,
+                height: 45,
+                width: 45,
+                position: "absolute",
+                zIndex: 999 - index,
```
Comment (tan-x) [inline pubio/components/Cards.js]: loving this styling logic

File: pubio/components/CarouselCards.js (hunk 1 above)
Comment (andrewvenson) [reply pubio/components/CarouselCards.js:21]: 😂 I’ve got no patience for them

File: pubio/components/screens/Profileimages.js [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+              style={styles.avatars}
+              size={100}
+              source={require("../../assets/profileimages/batman.jpg")}
+            />
+          </TouchableOpacity>
+          <TouchableOpacity>
+            <Avatar.Image
+              style={styles.avatars}
+              size={100}
+              source={require("../../assets/profileimages/goku.jpg")}
```
Comment (tan-x) [inline pubio/components/screens/Profileimages.js:89]: YES

File: pubio/components/DrawerContent.js [hunk 4]
```diff
@@ -25,6 +25,95 @@ export default function DrawerContent(props) {
...
 
+  var image = crawlcontext[2].profile;
+
+  const [profileImage, setProfilemage] = useState({
+    url: "",
+  });
+
+  // set current profile image on mount
+  useEffect(() => {
+    switch (image) {
```
Comment (tan-x) [inline pubio/components/DrawerContent.js:36]: clean

File: pubio/components/Cards.js [hunk 5]
```diff
@@ -23,49 +23,225 @@ export default function Card(props) {
...
-                  shadowRadius: 3.84,
-                  elevation: 8,
-                }}
-                source={{
-                  uri: user.profile,
-                }}
-                size={45}
-              />
-            );
+            switch (user.profile) {
```
Comment (tan-x) [inline pubio/components/Cards.js:30]: maybe use switch case on a source variable to dry this out?

Comment (andrewvenson) [reply pubio/components/Cards.js:30]: I was on that same line of thought, but unfortunately the require method doesn't take any variables. It has to be strictly a string 🙄

Comment (andrewvenson) [reply pubio/components/Cards.js:30]: we may be able to dry it up though, we'll have to brainstorm @tan-x

**Pull request merged**